app/models/concerns/curation_queue.rb

```diff
@@ -30,10 +30,12 @@ def notify_curators
   end
 
   def from_unverified_or_rejected?
+    return true unless user # no user set is about the same to me as an unverified user
+
```

Comment:
Is this needed? I think `user` is a required field on all curated types

Reply:
I ran into issues locally where there was no user set, which caused a crash. I think in general it is nice to be a little resilient to data quality issues.

Reply:
I think it would be better to address why the user was nil at that point, because I imagine a nil user will cause numerous other issues, and should be enforced by the association/validation

Reply:
Alright, will remove these safeties.